src/Test/Utilities/Portable/Assert/ConditionalFactAttribute.cs

```diff
@@ -205,6 +205,12 @@ public class DesktopOnly : ExecutionCondition
         public override string SkipReason => "Test not supported on CoreCLR";
     }
 
+    public class DesktopClrOnly : ExecutionCondition
```

Comment:
This looks like a duplicate of `WindowsDesktopOnly`

Reply:
Are the conditionals functionally equivalent? They look very different. Windows desktop can't be Mono on a Windows machine? If it really means '.net framework on windows only' it'd be good to rename it so it's obvious that it means that.

Reply:
Sync'd offline. This is not a duplicate. 

WindowsDesktopOnly - Microsoft Desktop Clr which is implicitly windows
DesktopClrOnly - Any desktop (mono or microsoft) which can be other OS

---
In reply to: [220985801](https://github.com/dotnet/roslyn/pull/30166#discussion_r220985801) [](ancestors = 220985801)